### ISSUE TITLE
[bug] MC 1.14.4 Amces won't save modifiers if you have optifabric enabled.

### ISSUE THREAD

**Issue by Panderine** [Siphalor/mouse-wheelie#58]
**Version**
MC 1.14.4  tested with latest MW available also with stand-alone amecs 1.2.7
**Describe the bug**
If you have optifabric enabled (to use optifine) custom controls that use modifiers (alt, ctrl, etc) won''t be saved to config the next time you restart MC.

**To Reproduce**
1. Add both mods
2. define a custom control key with some modifier 
3. Restart minecraft
4. custom control key is not there anymore.

**Expected behavior**
Saved custom control key modifications with modifiers.

**Log**
I guess this is the relevant part:
 - Mod amecs conflicts with mod {optifabric @ [*]}, but the conflicting version is present: 0.7.1!

**Additional context**

**Comment by Siphalor** [comment]
Thanks for reporting this :)

The related issue on Amecs' end is Siphalor/amecs-api#1.
**TL;DR:** I would love to resolve this problem but because of the invasive fashion of Optifine's code and because Optifine is closed source I won't be able to do it.

All conversation should be done there I'll keep this open just for reference for MW users.

**Comment by Siphalor** [comment]
I'm gonna finally close this as out of scope since the Amecs issue is the way to go :)
